src/providers/proxmoxve/cloudconfig.rs

```diff
@@ -33,26 +34,15 @@ pub struct ProxmoxVECloudMetaData {
 #[derive(Debug, Deserialize)]
 pub struct ProxmoxVECloudUserData {
     pub hostname: String,
-    pub manage_etc_hosts: bool,
-    pub fqdn: String,
-    pub chpasswd: ProxmoxVECloudChpasswdConfig,
-    pub users: Vec<String>,
-    pub package_upgrade: bool,
```

Comment:
What is it OK to delete this? Was nothing using it? 

Maybe in a separate first commit we should delete the things that are unused first?

Reply:
Yes they are not used and it can cause serde errors if they are not present. Specifically package_upgrade if missing from the YAMLs it would cause afterburn to fail.